portal/src/components/Materials/Materials.component.js

```diff
@@ -27,10 +27,7 @@ export default {
       type: String,
       default: 'delete'
     },
-    value: {
-      // type: Array,
-      // default: []
-    }
+    value: {}
```

Comment:
wat is het type dan?

Reply:
ik kom er niet uit. ik breng het terug naar wat het was

Reply:
het is soms een object en soms een array